toast/toast.css

```diff
@@ -0,0 +1,36 @@
+@import "../buttons/buttons.css";
+
+.container{
+    padding: 1rem;
+    gap: 2rem;
+}
+.component-container{
+    padding: 2rem 4rem 2rem 2rem;
+}
+.component-details{
+    padding: 1rem;
+}
+#myToast {
```

Comment:
Suggestion: Don't use id for styling as it will be problematic when you will have to override this css for your React apps